Sources/MapboxNavigation/NavigationMapView+VanishingRouteLine.swift

```diff
@@ -236,28 +239,23 @@ extension NavigationMapView {
 
                 let lineString = feature.geometry.value as? LineString
                 guard let distance = lineString?.distance() else { return gradientStops }
+                let stopGap = max(min(GradientCongestionBlendingDistance, distance * 0.1) / routeDistance, 0.0000000000000002)
```

Comment:
Is it possible to avoid this magic constant and use `Double.leastNonzeroMagnitude` instead?

Reply:
Yes, sure, I'll test to see whether it could be accepted by the style expression. Thanks for the suggestion.

Reply:
`Double.leastNonzeroMagnitude` could not be accepted and recognized with the style expression, and would lead to a blending effect without the distance control. The `0.0000000000000002` here is almost the difference of the `CGFloat.nextDow` different in Double, which would give a short blending effect is the congestion level is pretty short.